app/models/parsers/products/maximum_out_of_pockets_parser.rb

```diff
@@ -23,8 +23,8 @@ def to_hash
           name: name.gsub(/\n/, '').strip,
           in_network_tier_1_individual_amount: in_network_tier_1_individual_amount.gsub(/\n/, '').strip,
           in_network_tier_1_family_amount: in_network_tier_1_family_amount.gsub(/\n/, '').strip,
-          in_network_tier_2_individual_amount: in_network_tier_2_individual_amount.present? ? in_network_tier2_individual_amount.gsub(/\n/, '').strip : '',
-          in_network_tier_2_family_amount: in_network_tier_2_family_amount.present? ? in_network_tier2_family_amount.gsub(/\n/, '').strip : '',
+          in_network_tier_2_individual_amount: in_network_tier_2_individual_amount.present? ? in_network_tier_2_individual_amount.gsub(/\n/, '').strip : '',
+          in_network_tier_2_family_amount: in_network_tier_2_family_amount.present? ? in_network_tier_2_family_amount.gsub(/\n/, '').strip : '',
```

Comment:
can we support both here - with underscore & without underscore.

Reply:
The elements are defined with underscore, if we want to support without underscore then we have to define elements without underscore.